projects/1.19/assets/quark/quark/lang/zh_cn.json

```
@@ -1,1296 +1,3379 @@
+	"quark.config.experimental.adjustable_chat.desc": "可以调节聊天栏的水平/垂直偏移。",
+	"quark.config.experimental.adjustable_chat.horizontal_shift.name": "水平偏移",
+	"quark.config.experimental.adjustable_chat.vertical_shift.name": "垂直偏移",
+
+	"quark.config.experimental.climate_control_remover.name": "气候控制移除",
+	"quark.config.experimental.climate_control_remover.disable_continentalness.name": "禁用大陆性",
+	"quark.config.experimental.climate_control_remover.disable_continentalness.desc": "选择要生成的生物群系时，禁用「大陆性」的比较。\n警告：启用此项后，海洋可能会变得更像河流。",
+	"quark.config.experimental.climate_control_remover.disable_depth.name": "禁用深度",
+	"quark.config.experimental.climate_control_remover.disable_depth.desc": "选择要生成的生物群系时，禁用「深度」的比较。\n警告：启用此项后，洞穴生物群系可能会出现在不太常见的高度。",
+	"quark.config.experimental.climate_control_remover.disable_erosion.name": "禁用侵蚀度",
+	"quark.config.experimental.climate_control_remover.disable_erosion.desc": "选择要生成的生物群系时，禁用「侵蚀度」的比较。\n警告：启用此项后，可能会产生非常极端的高度差异，并将使末地更加混乱。",
+	"quark.config.experimental.climate_control_remover.disable_humidity.name": "禁用湿度",
+	"quark.config.experimental.climate_control_remover.disable_humidity.desc": "选择要生成的生物群系时，禁用「湿度」的比较。",
+	"quark.config.experimental.climate_control_remover.disable_offset.name": "禁用偏移",
+	"quark.config.experimental.climate_control_remover.disable_offset.desc": "选择要生成的生物群系时，禁用「偏移」的比较。\n警告：启用此项后，较稀有的下界生物群系会更常见。",
+	"quark.config.experimental.climate_control_remover.disable_temperature.name": "禁用温度",
+	"quark.config.experimental.climate_control_remover.disable_temperature.desc": "选择要生成的生物群系时，禁用「温度」的比较。",
+	"quark.config.experimental.climate_control_remover.disable_weirdness.name": "禁用奇异性",
+	"quark.config.experimental.climate_control_remover.disable_weirdness.desc": "选择要生成的生物群系时，禁用「奇异性」的比较。\n警告：启用此项后……好吧，可能使世界变得奇异。",
+
+	"quark.config.experimental.enchantment_predicates.name": "魔咒谓词",
+	"quark.config.experimental.enchantment_predicates.desc": "针对「要注册的魔咒」中定义的魔咒，此功能为「要更改的物品」中定义的物品生成资源包物品模型谓词。\n\n示例：如果将「minecraft:silk_touch」添加到「要注册的魔咒」中，将「minecraft:netherite_pickaxe」添加到「要更改的物品」中，那么名为「quark_has_enchant_minecraft_silk_touch」的谓词将可用于 netherite_pickaxe.json 物品模型，该谓词的值等于魔咒等级。",
+	"quark.config.experimental.enchantment_predicates.enchantments_to_register.name": "要注册的魔咒",
+	"quark.config.experimental.enchantment_predicates.items_to_change.name": "要更改的物品",
+
+	"quark.config.experimental.enchantments_begone.name": "魔咒禁用",
+	"quark.config.experimental.enchantments_begone.enchantments_to_begone.name": "要禁用的魔咒",
+
+	"quark.config.experimental.game_nerfs.name": "游戏削弱",
+	"quark.config.experimental.game_nerfs.disable_falling_block_dupe.name": "禁用掉落方块复制",
+	"quark.config.experimental.game_nerfs.disable_falling_block_dupe.desc": "使下落的方块无法（利用维度穿梭）复制。",
+	"quark.config.experimental.game_nerfs.disable_ice_roads.name": "禁用冰路",
+	"quark.config.experimental.game_nerfs.disable_ice_roads.desc": "使船无法在冰上滑行。",
+	"quark.config.experimental.game_nerfs.disable_iron_farms.name": "禁用刷铁机",
+	"quark.config.experimental.game_nerf
```

```suggestion
	"quark.config.oddities.totem_of_holding.dark_souls_mode.desc": "设为false后，玩家死亡时，已存在但未取走的图腾不会自毁。",
```
得，就到这里，看看明天能不能给这个PR打上approved

---

我觉得没啥意义，`已存在` 和 `未取走` 等价，且游戏中也不存在所谓 `不存在` 或 `已取走` 的图腾，即使所谓 `已取走` 的图腾仍在游戏中存在，那它的自毁也没有意义，完全算不上是惩罚。
如果是为了合理地把 `再次` 一词去掉，就强行加上一串谜之定语的话，我觉得不太行。

---

这些定语不是用来描述图腾，其实`是用来给玩家提示的，
```suggestion
	"quark.config.oddities.totem_of_holding.dark_souls_mode.desc": "设为false后，玩家死亡时，未取走的图腾不会自毁。",
```

---

我说 即使所谓 已取走 的图腾仍在游戏中存在，那它的自毁也没有意义，完全算不上是惩罚 还有一个意思就是：
玩家看到图腾会自毁，肯定会想：那我没取走的图腾里面物品不全没了？（而不会考虑到已经取过烟消云散的图腾，因为没有意义）
不过算了，加个未取走的也不长，就是你能把再次加上吗？这样才有种每死一次都要赶紧拿回图腾的感觉
话说你这下面的 Suggested change 全 buggy 了，什么个情况啊？我都不太敢 Commit

---

```suggestion
	"quark.config.oddities.totem_of_holding.dark_souls_mode.desc": "设为false后，玩家死亡时，未取走的图腾不会自毁。",
```
我怀疑显示出错是自动换行的原因，但是行号是对的，只是内容不对。
放心提交，出错了可以 revert commit